third_party/nvfuser/csrc/lower_unroll.h

```diff
@@ -68,8 +69,6 @@ class TORCH_CUDA_CU_API UnrollPass : kir::ExprMutator {
     return expr_replacement_map_;
   }
 
-  using OptOutDispatch::handle;
```

Comment:
This should be `using kir::ExprMutator::handle;`, but just removing it results in a build error for me (using clang):

```
/raid/tmp/nmaruyama/debug1/third_party/nvfuser/csrc/lower_unroll.h:74:8: error: 'nvfuser::UnrollPass::handle' hides overloaded virtual function [-Werror,-Woverloaded-virtual]
  void handle(Expr* expr) final;
       ^
/raid/tmp/nmaruyama/debug1/third_party/nvfuser/csrc/kernel_ir_dispatch.h:36:16: note: hidden overloaded virtual function 'nvfuser::kir::IrVisitor::handle' declared here: type mismatch at 1st parameter ('nvfuser::kir::IfThenElse *' vs 'nvfuser::Expr *')
  virtual void handle(IfThenElse*) override;
```

Reply:
This doesn't make sense to me. But I will not argue with a compiler about which is correct, so I just added it back.